extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -87,6 +93,15 @@ class AmpStickyAd extends AMP.BaseElement {
     }
   }
 
+  removeOnClickListener_() {
```

Comment:
But what happens with the `amp-sticky-ad`? It should probably be removed, in which case the button will be removed and so will all of its listeners. Thus, you need less ceremony with removal. That is, of course, if you indeed intend to remove the element.